src/test/java/com/exasol/adapter/document/DocumentAdapterIT.java

```diff
@@ -0,0 +1,109 @@
+package com.exasol.adapter.document;
+
+import static com.exasol.adapter.document.UdfEntryPoint.*;
+import static com.exasol.matcher.ResultSetStructureMatcher.table;
+import static com.exasol.matcher.TypeMatchMode.NO_JAVA_TYPE_CHECK;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import java.io.FileNotFoundException;
+import java.nio.file.Path;
+import java.sql.*;
+import java.util.Map;
+import java.util.concurrent.TimeoutException;
+import java.util.logging.Logger;
+
+import org.apache.maven.it.VerificationException;
+import org.apache.maven.it.Verifier;
+import org.junit.jupiter.api.*;
+
+import com.exasol.bucketfs.BucketAccessException;
+import com.exasol.dbbuilder.dialects.exasol.*;
+import com.exasol.dbbuilder.dialects.exasol.udf.UdfScript;
+import com.exasol.errorreporting.ExaError;
+import com.exasol.exasoltestsetup.ExasolTestSetup;
+import com.exasol.exasoltestsetup.testcontainers.ExasolTestcontainerTestSetup;
+import com.exasol.udfdebugging.UdfTestSetup;
+
+class DocumentAdapterIT {
+    private static final String MY_VIRTUAL_SCHEMA = "MY_VIRTUAL_SCHEMA";
+    private static final String ADAPTER_NAME = "FIXED_DATA_ADAPTER";
+    private static final String MOCK_ADAPTER_JAR = "mock-adapter.jar";
+    private static final Logger LOGGER = Logger.getLogger(DocumentAdapterIT.class.getSimpleName());
+    private static ExasolTestSetup testSetup;
+    private static ExasolObjectFactory exasolObjectFactory;
+    private static ConnectionDefinition nullConnection;
+    private static AdapterScript adapterScript;
+
+    @BeforeAll
+    static void beforeAll() throws SQLException, BucketAccessException, TimeoutException, FileNotFoundException {
+        testSetup = new ExasolTestcontainerTestSetup();
+        final UdfTestSetup udfTestSetup = new UdfTestSetup(testSetup);
+        exasolObjectFactory = new ExasolObjectFactory(testSetup.createConnection(),
+                ExasolObjectConfiguration.builder().withJvmOptions(udfTestSetup.getJvmOptions()).build());
+        buildMockAdapter();
+        final ExasolSchema adapterSchema = exasolObjectFactory.createSchema("ADAPTER");
+        testSetup.getDefaultBucket().uploadFile(Path.of("test-project", "mock-project", "target", MOCK_ADAPTER_JAR),
```

Comment:
```suggestion
        testSetup.getDefaultBucket().uploadFile(Path.of("test-project/mock-project/target", MOCK_ADAPTER_JAR),
```
This is more readable.

Reply:
is it also platform independant?

Reply:
Yes. I always used it also under windows.